src/routes/index.tsx

```diff
@@ -81,7 +82,8 @@ const Routes = (): React.ReactElement => {
         getNetworkRootRoutes().map(({ chainId, route }) => (
           <Route
             key={chainId}
-            path={route}
+            // /xdai also sets chainId correctly
+            path={chainId === CHAIN_ID.GNOSIS_CHAIN ? [route, '/xdai'] : route}
```

Comment:
I would add a redirect instead of this

Reply:
I would assume to `/gnosis-chain`? It needs to set the `chainId` too, hence setting it here.